docs/source/development/style-guide.rst

```diff
@@ -0,0 +1,391 @@
+``imap_processing`` Style Guide
+=============================
+
+This document serves as a style guide for all ``imap_processing`` software development. Any potential contribution to
+the ``imap_processing`` repository should be checked against this guide, and any violation of the guide should be fixed
+before the code is committed to the ``main`` or ``dev`` branches.
+
+
+.. _tl;dr:
+tl;dr
+-----
+
+The following is a short version of this style guide to be used as a quick reference. Further details about each of
+these items are provided below in the guide.
+
+1. Use a :ref:`forking workflow <git-and-github-workflow>` for git/GitHub contributions.
```

Comment:
Nitpick/in case you didn't know - you can replace the numbers (ie `1.`) with the pound sign (`#.` and it will automatically number things. So someone could add a point 4 without needing to update point 5, 6, etc...

Reply:
ah, cool!